ebu_tt_live/documents/ebuttd.py

```diff
@@ -74,9 +74,11 @@ def _get_bds(self):
                 namespace_prefix_map=bindings.namespace_prefix_map
             )
 
-    def get_xml(self):
+    def get_xml(self, indent=None, newl=None):
```

Comment:
Is this meant to remove the extra lines from the output ebutt3 doc?

Reply:
No, not by itself, but it does do that if `get_xml()` is called with `indent='', newl=''`. This code keeps the existing behaviour if those parameters are omitted.